spec/features/cart/user_can_add_items_to_cart_spec.rb

```diff
@@ -11,59 +11,81 @@
   end
 
   scenario "the cart counter says 1 after adding the first item" do
-    within "#item_#{pants.id}" do
-      click_on "Add to cart"
-    end
+    add_one_item_to_cart(pants)
 
```

Comment:
I just want to make sure we are still testing that each item will have it's own "Add to cart" button that can be clicked. We just have to test it once in this feature test.